sonar-pmd-plugin/src/main/java/org/sonar/plugins/pmd/PmdConstants.java

```diff
@@ -42,17 +42,17 @@ public final class PmdConstants {
     /**
      * Default value for property {@link #JAVA_SOURCE_VERSION}.
      */
-    public static final String JAVA_SOURCE_VERSION_DEFAULT_VALUE = "24";
+    public static final String JAVA_SOURCE_VERSION_DEFAULT_VALUE = "21";
```

Comment:
Should be 25. This makes it accept all syntax including 25.

Reply:
ok, I thought it was default version for analysis :-)